spec/checkPullRequestLabelsSpec.js

```diff
@@ -117,6 +127,27 @@ describe('Pull Request Label Check', () => {
         expect(github.issues.addAssignees).not.toHaveBeenCalled();
       });
 
+    it('should not assign project owner if they are already assigned',
```

Comment:
I don't think it's possible to assign a person when they are already assigned to an issue.

Reply:
Yes, my bad. The purpose is to ensure that comment is not added. Updated.